packages/sdk/src/__tests__/getGenerators/getGenerators.test.ts

```diff
@@ -747,84 +747,6 @@ describe('getGenerators', () => {
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
   })
 
-  test('fail if mongoDb not found in previewFeatures - prisma-client-js - mongodb', async () => {
-    expect.assertions(5)
-    const aliases = {
-      'predefined-generator': {
-        generatorPath: generatorPath,
-        outputPath: __dirname,
-      },
-    }
-
-    try {
-      await getGenerators({
-        schemaPath: path.join(__dirname, 'missing-mongoDb-from-previewFeatures-client-js.prisma'),
-        providerAliases: aliases,
-        skipDownload: true,
-      })
-    } catch (e) {
-      expect(stripAnsi(e.message)).toMatchInlineSnapshot(`
-        "
-        In order to use the mongodb provider,
-        you need to set the mongodb feature flag.
-        You can define the feature flag like this:
-
-        generator client {
-            provider = \\"prisma-client-js\\"
-            previewFeatures = [\\"mongoDb\\"]
-        }
-
-        More information in our documentation:
-        https://pris.ly/d/prisma-schema
-        "
-      `)
-    }
-
-    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-  })
-
-  test('fail if mongoDb not found in previewFeatures - prisma-client-go - mongodb', async () => {
-    expect.assertions(5)
-    const aliases = {
-      'predefined-generator': {
-        generatorPath: generatorPath,
-        outputPath: __dirname,
-      },
-    }
-
-    try {
-      await getGenerators({
-        schemaPath: path.join(__dirname, 'missing-mongoDb-from-previewFeatures-client-go.prisma'),
-        providerAliases: aliases,
-        skipDownload: true,
-      })
-    } catch (e) {
-      expect(stripAnsi(e.message)).toMatchInlineSnapshot(`
-        "
-        In order to use the mongodb provider,
-        you need to set the mongodb feature flag.
-        You can define the feature flag like this:
-
-        generator client {
-            provider = \\"prisma-client-js\\"
-            previewFeatures = [\\"mongoDb\\"]
-        }
-
-        More information in our documentation:
-        https://pris.ly/d/prisma-schema
-        "
-      `)
-    }
-
-    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-  })
```

Comment:
Might be worth keeping around for future providers. Makes it easier to remember this is a useful test I would say.